macos/package_files/preinstall.sh

```diff
@@ -11,16 +11,17 @@
 
 DIR="/Library/Ossec"
 
-if [ ! -d ${DIR} ]; then
-    launchctl setenv WAZUH_PKG_UPGRADE false
-else
-    launchctl setenv WAZUH_PKG_UPGRADE true
+if [ -d ${DIR} ]; then
+    rm -f ${DIR}/WAZUH_PKG_UPGRADE
+    rm -f ${DIR}/WAZUH_RESTART
+    touch ${DIR}/WAZUH_PKG_UPGRADE
```

Comment:
The WAZUH_PKG_UPGRADE is removed in the postinstall.sh script, this file should not exist in the preinstall.sh 

If this is necessary is needed to add a check if the file exists